src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    int areaCalculation();
```

Comment:
method names should be verbs, or verb phrases, nouns are for class names
```suggestion
    int calculateArea();
```